schemas/aws/terraform-resource-2.yml

```diff
@@ -961,6 +976,8 @@ oneOf:
           - ips
         - required:
           - openshift_service
+          - protocol
+          - protocol_version
```

Comment:
This makes those new fields required when `openshift_service` is set. Is that on purpose? Don't we want to rely on current defaults?

Reply:
@patjlm  remove the protocol and protocol_version to be rquired